src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -485,6 +489,10 @@ private void StartReadStream(StreamReader reader, ConcurrentQueue<string> dataBu
                     string line = reader.ReadLine();
                     if (line != null)
                     {
+                        if (DisallowWorkerCommands)
+                        {
+                            line = Regex.Replace(line, "##vso", "##DISALLOWED-vso", RegexOptions.IgnoreCase);
```

Comment:
Do we want to print like this or print an info message with the contents describing what happened and the original message? Not sure what is the best UX and what will help customers the most.

Reply:
Yeah, this doesn't feel quite right. Maybe replace the `##vso` with `**vso` and append a `##warning` line explaining what happened?

Reply:
I changed "##vso" to "**vso" but am unable to inject a warning message with my current understanding of the code base